src/equiv.js

```diff
@@ -59,6 +59,10 @@ QUnit.equiv = (function() {
 		return false;
 	}
 
+	function getRegExpFlags( regexp ) {
+		return regexp.flags || regexp.toString().match(/[gimuy]*$/)[0];
```

Comment:
@gibson042 
An error in jshint occurred when i made testcase for `u` option. It working this issue in [js](https://github.com/jshint/jshint/issues/2361) [hint](https://github.com/jshint/jshint/pull/2369). Can i use jshint except options in test code?